app/src/main/java/dev/arkbuilders/arkmemo/repo/versions/VersionsStorageRepo.kt

```
@@ -0,0 +1,4 @@
+package dev.arkbuilders.arkmemo.repo.versions
+
+class VersionsStorageRepo {
+}
```

detekt (1.23.8)

[warning] The class or object VersionsStorageRepo is empty.

(detekt.empty-blocks.EmptyClassBlock)